### PR TITLE
[Feature] Update Stats Section Gradient Wave

### PR DESCRIPTION
**Pull request by ABCodez**
Issue: #142
Branch: feature/142/update-stats-section-gradient-wave

Changes:
 - Stats section now has the same waves as the Figma design
 
image
image

Future updates (hopefully):
- Animate the waves to make this section look even better (I spent way too long trying to do this, so am taking a break from it 😿)

### CODE REVIEW

**Comment by juancwu** [comment]
Screenshot from 2024-03-03 22-43-47

On 2k resolution, the bottom part of the svg gets cropped.

File: src/components/InfoBanner/InfoBanner.css [hunk 1]
```diff
@@ -2,12 +2,17 @@
...
     display: flex;
-    justify-content: space-between;
+    justify-content: center;
     align-items: center;
     padding-left: 10%;
     padding-right: 10%;
     gap: 12em;
     font-size: 18px;
+
+    background-image: url('../../../public/InfobannerBackground.svg');
```
Comment (juancwu) [inline src/components/InfoBanner/InfoBanner.css]:
```suggestion
    background-image: url('/InfobannerBackground.svg');
```

These will make the resource be serves as a public static resource.

File: src/components/InfoBanner/InfoBanner.css [hunk 2]
```diff
@@ -2,12 +2,17 @@
...
+    background-size: cover;
+    width: 100%;
+    height: 500px;
```
Comment (juancwu) [inline src/components/InfoBanner/InfoBanner.css]: The fixed height here is what I think causes the cropping.

File: public/InfobannerBackground.svg
Comment (juancwu) [inline public/InfobannerBackground.svg:1]: Don't know if it would be better if we separated the top and bottom waves. Might allow bigger flexibility.

**Review by juancwu — CHANGES_REQUESTED** [review]

File: src/components/InfoBanner/InfoBanner.css (hunk 1 above)
Comment (ABCodez) [reply src/components/InfoBanner/InfoBanner.css]: Fixed in most recent commit!

File: public/InfobannerBackground.svg
Comment (ABCodez) [reply public/InfobannerBackground.svg:1]: I don't really think its necessary to have that? its only after ~2600px width where having that flexibility might matter more

Comment (juancwu) [reply public/InfobannerBackground.svg:1]: Sure, we can leave it as is. I am thinking we could add a button for like reporting bugs on the website lol.

**Review by juancwu — APPROVED** [review]
Looking :fire:

**Pull request merged**